docs/docsource/operation/XDM_GUI/location_management/adding_a_storage_location.rst

```diff
@@ -0,0 +1,278 @@
+.. _orbit_add_location:
+
+Adding a Storage Location
+=========================
+
+To add a storage location:
+
+#. From the Account view, select an account name to expose account information.
+
+   .. image:: ../../Graphics/xdm_ui_account_view.png
+      :align: center
+	      
+#. Click the **Locations** tab.
+
+   .. image:: ../../Graphics/xdm_ui_locations_tab.png
+      :align: center
+	      
+#. Click **Create Location**.
+
+#. The **ADD NEW STORAGE LOCATION** dialog displays:
+
+   .. image:: ../../Graphics/xdm_ui_add_new_storage_location.png
+      :align: center
+
+   a. Enter a location name in the **Location Name** field using
+      lowercase letters, numbers, and dashes.
+
+      .. note::
+
+         Capital letters, spaces, and punctuation and diacritical
+         marks will result in an error message.
+
+   b. Select a location type from the **Location Type** pull-down menu.
+      You can choose:
+
+      * Amazon S3
+      * Ceph RADOS Gateway
+      * DigitalOcean Spaces
+      * Google Cloud Storage
+      * Microsoft Azure Blob Storage
+      * NFS Mount
+      * Scality RING with S3 Connector
+      * Scality RING with sproxyd Connector
+      * Scality XCore
+      * Wasabi
+      * A Zenko local filesystem
```

Comment:
Use product for Zenko here?

Reply:
I don't know. Neither does Product. This is one case where I've backed away from the full-on sed -i 's/Zenko/XDM/g' approach: I want to verify that the filesystem is actually capable of being renamed "XDM" easily before I assume that we can change all references to it with a wave of the marketing wand.